vnext/Microsoft.ReactNative.Cxx/NativeModules.h

```diff
@@ -627,7 +627,7 @@ struct ModuleSyncMethodInfo<TResult (TModule::*)(TArgs...) noexcept>
       ArgTuple typedArgs{};
       ReadArgs(argReader, std::get<I>(typedArgs)...);
       TResult result = (module->*method)(std::get<I>(std::move(typedArgs))...);
-      WriteArgs(argWriter, result);
+      WriteValue(argWriter, result);
```

Comment:
>WriteValue [](start = 6, length = 10)

I would expect that we have tests for it and if it did not work, they must not pass. I wonder what scenario is currently broken.